autokeras/hypermodels/wrapper.py

```diff
@@ -1,3 +1,20 @@
+import numpy as np
+from tensorflow.keras import backend as K
+from tensorflow.keras.activations import relu
+from tensorflow.keras.layers import Activation
```

Comment:
we use `from tensorflow.keras import layers`.
Then use `layers.Activation`.
The same for the rest of the imports.